data/json/items/armor/storage.json

```diff
@@ -3405,5 +3405,66 @@
     "armor": [
       { "encumbrance": [ 30, 250 ], "coverage": 70, "covers": [ "torso" ], "specifically_covers": [ "torso_hanging_back" ] }
     ]
+  },
+  {
+    "id": "seabag",
+    "type": "ARMOR",
+    "//": "based off of USMC seabag",
+    "name": { "str": "military seabag" },
+    "description": "A massive, watertight, olive drab duffel bag used to move soft goods like uniforms when being redeployed.  While it has straps to be worn like a backpack, it is extremely cumbersome to do so.",
+    "weight": "1000 g",
+    "volume": "4 L",
```

Comment:
```suggestion
    "volume": "80 L",
```

Reply:
Doesn't this make the seabag occupy 81L of space even if it's empty and collapsed? The seabag is just made of cloth.